Tools/_sunrise/changelog/changelog_actions.py

```
@@ -150,20 +161,150 @@ def write_validation_summary(title: str, content: str) -> None:
+            return sorted(set(paths))
+        page += 1
+
+
+def load_repository_file(path: str, ref: str) -> str:
+    repository = os.environ.get("GITHUB_REPOSITORY", "")
+    response = github_request(
+        f"/repos/{repository}/contents/{quote(path, safe='/')}",
+        {"ref": ref},
+    )
+    if not isinstance(response, dict) or not isinstance(response.get("content"), str):
+        raise RuntimeError(f"GitHub API не вернул содержимое файла {path}")
```

Ruff (0.16.2)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)